docs/getting-started/installation.md

```diff
@@ -8,13 +8,17 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ```
 
-There are various ways to install Chia. The best method depends on what you intend to do.
+There are various ways to install Chia, with the best method depending on what you intend to do:
 
-If you plan to use Chia as a wallet or to run a farm on a personal computer, we recommend installing the GUI from our [official downloads page](https://www.chia.net/downloads). The best way to install the GUI on Linux is the command line, [as described below](#using-the-cli). The GUI is the simplest way to interact with the Chia client.
+- If you simply wish to use the Chia wallet, or to run a farm on a single personal computer, then we recommend installing the GUI from our [official downloads page](https://www.chia.net/downloads) for Windows and MacOS, and for Linux users to install the package [as described below](#using-the-cli). The GUI is the simplest way to interact with the Chia client and ideal for most non-developer use cases.
 
-However, if you intend to run a Chia full node on a server and connect to it programmatically using the [RPC interface](/rpc), the best method would be to install and run Chia on the command line.
+- If you intend to run a dedicated Chia full node on a server and connect to it programmatically using the [RPC interface](/rpc), the best method would be to install and run Chia via the command line on a proper server environment.
 
-Finally, if you plan on making contributions to the source code or doing [Chialisp](https://chialisp.com) development, we recommend installing Chia from source. This gives the highest level of flexibility.
+- If you inded to do [Chialisp](https://chialisp.com) development or build projects that leverage Chia, you have the options of either using an installer (the recomended pattern), or installing from source.
```

Comment:
indend -> intend

Reply:
and to think I dumped this into a separate tool to spell check TWICE...

thanks!

Reply:
corrected in updated PR